db/config.js

```diff
@@ -0,0 +1,19 @@
+const path = require('path');
+
+try {
+  require('dotenv').load({ path: path.resolve(__dirname, '..', '.env') });
+} catch (e) { }
+
+const config = {
+  username: process.env.PGUSER,
+  password: process.env.PGPASSWORD,
+  database: process.env.PGDATABASE,
+  host: process.env.PGHOST,
+  dialect: 'postgres',
+};
+
+module.exports = {
+  development: config,
+  test: config,
+  production: config,
```

Comment:
Oh I see what you mean, we just change our .env depending on what we want. Do you want to make any changes to this now? Otherwise I'm good to merge